build.sbt

```diff
@@ -46,18 +46,14 @@ lazy val core = crossProject(JSPlatform, JVMPlatform)
   .settings(
     name := "log4cats-core",
     libraryDependencies ++= Seq(
-      "org.typelevel" %%% "cats-core" % catsV
+      "org.typelevel" %%% "cats-core"       % catsV,
+      "org.typelevel" %%% "cats-effect-std" % catsEffectV
```

Comment:
Just to keep the initial goal of having the core module CE-less, can we introduce an internal interface for UUIDGen and its implementation within the slf4j module?

Reply:
Can we instead create a new module that is _explicitely_ dependent on CE? And move the correct version of this there?

I feel like re-defining a UUIDGen equivalent leads to a lot of confusion and bad UX. Instead having a `log4cats-core` package that is honest about its limitations, and an extension `logs4cats-ce` (naming is hard) that overcomes aforementioned limitations might be better.

Although not entirely sure how to pull this off in a bincompat _and_ user friendly way.

Reply:
I agree, adding a "`log4cats-effect`" would be better for UX, but still not great for UX+bincompat.

Just so that we are clear: what are the arguments against just have a CE dependency in the core module?

Reply:
> what are the arguments against just have a CE dependency in the core module?

Very good question! I personally don't know since I probably missed those discussions during my absence. :see_no_evil: Personally I think it's ok to have CE in `core`

Reply:
I can only guess, but perhaps the reason is to apply the Rule of Least Power. But once again, I'm not sure if this is a thing still.

Reply:
I would say it is a thing still :) The problem is the repo is currently applying the principle of too-little-power. This is a course-correction.